rdkit/Chem/MolKey/MolKey.py

```diff
@@ -314,45 +303,40 @@ def _get_bad_mol_identification_string(ctab, stereo_category, extra_stereo):
   else:
     pass
   if stereo_category:  # add xtra info if available
-    key_string = 'ST={0}'.format(stereo_category)
-    pieces.append(key_string)
+    pieces.append(f'ST={stereo_category}')
   if extra_stereo:  # add xtra info if available
-    key_string = 'XTR={0}'.format(extra_stereo)
-    pieces.append(key_string)
-  key_string = '/'.join(pieces)
-  return key_string
+    pieces.append(f'XTR={extra_stereo}')
+  return '/'.join(pieces)
 
 
 def _identify(err, ctab, inchi, stereo_category, extra_structure_desc=None):
   """ Compute the molecule key based on the inchi string,
     stereo category as well as extra structure
     information """
   key_string = _get_identification_string(err, ctab, inchi, stereo_category, extra_structure_desc)
-  if key_string:
-    return "{0}|{1}".format(
-      MOL_KEY_VERSION, base64.b64encode(hashlib.md5(key_string.encode('UTF-8')).digest()).decode())
-  else:
+  if not key_string:
     return None
+  string = base64.b64encode(hashlib.md5(key_string.encode('UTF-8')).digest()).decode()
+  return f"{MOL_KEY_VERSION}|{string}"
```

Comment:
Don't use `string` as a variable name. There is a Python package called `string`. Call it e.g. `hash_key`

Reply:
OK, I got your idea! I would fix it in the next patch. Thank you for your review.